src/app/core/services/scheduling.service.ts

```diff
@@ -319,7 +306,8 @@ export class SchedulingService {
       index: index,
       completed: false,
       reportedCompletion: false,
-      timestamp: taskDate.getTime(),
+      // NOTE: Plus one hour added for consistency, but must be fixed in protocol.
+      timestamp: taskDate.getTime() + getMilliseconds({ hours: 1 }),
```

Comment:
I think maybe we should fix this now. It must not be too much work updating the protocols but we must make sure that the protocol update aligns with the app release.

Reply:
I can do this. Let me know when the changes must go live